bg/uni/sofia/fmi/mjt/sentiment/Reviewer.java

```diff
@@ -0,0 +1,49 @@
+package bg.uni.sofia.fmi.mjt.sentiment;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class Reviewer {
```

Comment:
The name of the class doesn't correspond to it's content.
Do you really need all the methods to be static?